package.json

```diff
@@ -55,7 +55,7 @@
     "yarn": "^1.16.0"
   },
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "start": "adb reverse tcp:8081 tcp:8081 && node node_modules/react-native/local-cli/cli.js start",
```

Comment:
If you have no connected Android devices or open emulators, this will fail. It might be better to have a separate command for Android, or to use a single '&' potentially?